.github/workflows/integration-flows.yaml

```diff
@@ -21,11 +21,11 @@ jobs:
 
             - name: Install nango CLI
               run: |
-                # compile, install CLI and verify it can run
-                npm run i
-                npm run ts-build
+                # Build, install CLI and verify it can run
+                npm install
+                npm run build -w @nangohq/node -w @nangohq/shared -w nango
                 npm install -g ./packages/cli
-                nango version
+                NANGO_CLI_UPGRADE_MODE=ignore nango version
```

Comment:
Maybe do this with the debug flag so we get even more confirmation of what Nango it is using 

nango version —debug